modules/ai/module.php

```diff
@@ -1546,4 +1549,49 @@ private function update_product_gallery( $product, ?int $image_to_remove, ?int $
 		$product->set_gallery_image_ids( $gallery_image_ids );
 		$product->save();
 	}
+
+	private function should_display_create_with_ai_banner() {
+		return true;
+		// Check if the site has more than 10 pages with Elementor
+		$elementor_pages = new \WP_Query( [
+			'post_type' => 'page',
+			'post_status' => 'publish',
+			'fields' => 'ids',
+			'meta_key' => '_elementor_edit_mode',
+			'meta_value' => 'builder',
+			'posts_per_page' => self::MIN_PAGES_FOR_CREATE_WITH_AI_BANNER + 1,
+		] );
+
+		if ( $elementor_pages->post_count > self::MIN_PAGES_FOR_CREATE_WITH_AI_BANNER ) {
+			return false;
+		}
+
+		// Check if a custom kit is applied (not the default one)
+		$previous_kit_id = Plugin::$instance->kits_manager->get_previous_id();
```

Comment:
Maybe extract to a function (in utils)
Also check utils , we might already have this check